src/pid.cpp

```diff
@@ -400,9 +400,6 @@ double Pid::getCurrentCmd()
 
 void Pid::getCurrentPIDErrors(double *pe, double *ie, double *de)
 {
-  // Get the gain parameters from the realtime buffer
-  Gains gains = *gains_buffer_.readFromRT();
```

Comment:
I remember this frmo the previous PR. Reading the code suggests that really nothing happens here, right?

Could you please put this change into a different commit? It doesn't belong with the LeadLag stuff. Could still pass within the same PR as a minor fix.

Reply:
Fixed ;)